packages/client/fixtures/blog/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "strict": true,
-    "target": "ES2020",
+    "target": "ES2021",
```

Comment:
Given that we don't include them consistently and that they are a changing list, I'm inclined to say that we just omit the comments in the other `tsconfig` files, thoughts?